in-app-tutorials/getting-started/shared-tutorials/execute-with-kre.md

```diff
@@ -0,0 +1,25 @@
+## Execute tests with Katalon Runtime Engine
```

Comment:
How to open command prompt -> use can reuse the design from the onboarding tour, and also consider collapsible since this is quite long, and have 3 different OS (https://gist.github.com/pierrejoubert73/902cc94d79424356a8d20be2b382e1ab)